packs/st2/config.yaml

```diff
@@ -1,5 +1,14 @@
 ---
-  base_url: "http://localhost"
-  # auth_token will expire per token_ttl specifed in stackstorm
-  # configuration and will require a refresh.
-  auth_token: ""
+base_url: "http://localhost"
+# auth_token will expire per token_ttl specifed in stackstorm
+# configuration and will require a refresh.
+auth_token: ""
+
+### StackStorm S3 Information
+##  These settings are for a write-only S3 account used
+##  when sending information to StackStorm for troubleshooting/analytics
+s3:
+  access_key_id: "AKIAIDM5AFYIL6QTZUIQ"
+  secret_access_key: "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"
```

Comment:
Not at all. It's a write only access key, with only one IAM policy. Those can be public, for all I care. Worst someone can do is write to a single bucket. If it's abused, we'll address.